imagesearch/main.py

```diff
@@ -0,0 +1,68 @@
+"""
+Main training task.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+
+import argparse, os
+import tensorflow as tf
+
+from tensorflow.contrib.learn.python.learn import learn_runner
+
+from imagesearch.experiment import generate_experiment_fn
+
+def main():
+    "Entrypoint for training."
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        '--train-files',
+        help='Training files pattern globstring',
+        default='data/training/*.jpg')
+    parser.add_argument(
+        '--eval-files',
+        help='Evaluation files pattern globstring',
+        default='data/validation/*.jpg')
+    parser.add_argument(
+        '--job-dir',
+        help='Location to write checkpoints, summaries, and export models',
+        required=True)
+    parser.add_argument(
+        '--num-epochs',
+        help='Maximum number of epochs on which to train',
+        default=1,
+        type=int)
+    parser.add_argument(
```

Comment:
Since the train and eval batch sizes are the same should we consolidate the two?